src/components/Home.jsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchCryptos, getFormValue } from '../redux/balance/balanceSlice';
+import Headers from './Header';
+import Category from './Categories';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const [formValue, setFormValue] = useState({ searchValue: '' });
+
+  const {
+    cryptos, singleCrypto, loading, error,
+  } = useSelector((state) => state.cryptos);
+
+  useEffect(() => {
+    dispatch(fetchCryptos());
+  }, [dispatch]);
+
+  const handleChange = (e) => {
+    setFormValue(e.target.value);
+  };
+
+  const getValue = (e) => {
+    e.preventDefault();
+    if (formValue.trim() !== '' || formValue.searchValue !== '') {
+      document.querySelector('.submit').reset();
+      const toUpperCaseText = formValue.toUpperCase();
+      dispatch(getFormValue(
+        toUpperCaseText,
+      ));
+    }
+    setFormValue({ searchValue: '' });
+  };
+  return (
+    <section className="bg-blue">
+      <Headers title="CRYPTO, FOREX & COMMODITIES" />
+
+      <span className={loading ? 'bg-load' : ''}>{loading ? 'Loading...' : ''}</span>
+      <span>{error}</span>
+
+      <aside className="text-white ">
+        <div className="bg-image" />
+        <div className="bg-drop p-3">
+          <aside className="pt-2">
+            <h1>{singleCrypto.symbol}</h1>
+            <p>Symbol</p>
+          </aside>
+          <aside className="pt-2">
+            <h1>{singleCrypto.currency}</h1>
+            <p>Currency</p>
+          </aside>
+          <Link to={singleCrypto.symbol}>
+            <i className="fa-regular fa-circle-right cursor-pointer" />
+          </Link>
+
+        </div>
+      </aside>
+
+      <aside>
+        <form onSubmit={getValue} className="submit">
+          <input type="text" placeholder="e.g btcusd" name="searchValue" onChange={handleChange} />
+          <button type="submit" className="cursor-pointer">Search</button>
+        </form>
```

Comment:
- [ ] Great job with the project, I admire your effort so far, You've done well. I just wanted to point out something concerning the search. As it's a requirement to add a search functionality, so a user could search by an item name, I don't seem to see it working as intended. When a user searches for an item, it should show up what matches the search, It does not respond to the `OnChange` event 👍 . Please, try to fix this implementation to meet this part of the project requirement. :+1:


https://github.com/boniface193/react-final-capstone/assets/80458199/9e694a1b-3f4b-4754-80d9-fddfe9e46799